test/features/add_property/view/widgets/add_features_view_test.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hatspace/data/property_data.dart';
+import 'package:hatspace/features/add_property/view/widgets/add_features_view.dart'
+    as view;
+import '../../../../widget_tester_extension.dart';
+
+void main() {
+  testWidgets('[Add Features screen] Verify UI', (WidgetTester tester) async {
+    Widget widget = const view.AddFeaturesView();
+    const List<String> features = [
+      'Fridge',
+      'Washing machine',
+      'Swimming pool',
+      'Air conditioners',
+      'Electric stove',
+      'TV',
+      'Wifi',
+      'Security cameras',
+      'Kitchen',
+      'Portable fans'
+    ];
+
+    await tester.wrapAndPump(widget);
+    expect(find.byType(view.AddFeaturesView), findsOneWidget);
+    expect(find.text('Which features your place has?'), findsOneWidget);
+    final Finder featuresItem = find.ancestor(
+        of: find.byType(SvgPicture),
+        matching: find.byType(view.FeatureItemView));
+    final firstFeat =
+        tester.firstWidget<view.FeatureItemView>(featuresItem.first);
+    final lastFeat =
+        tester.firstWidget<view.FeatureItemView>(featuresItem.last);
+    expect(features.contains(firstFeat.feature.displayName), true);
+    expect(features.contains(lastFeat.feature.displayName), true);
+    expect(featuresItem, findsWidgets);
```

Comment:
Can u include the text for all feature names?

Reply:
And the number of available features as well. Make sure when new feature added, this test case should be failed

Reply:
@hophongit98 It will fail if new feature was added.

Reply:
@suesitran You want just the features that is currently being displayed on the screen ? Or all defined features ? Currently i'm using GridView.builder, so some features will not be rendered until user scroll down

Reply:
All features, scroll to check if you need to